demisto_sdk/commands/common/logger.py

```diff
@@ -93,39 +98,63 @@ def logging_setup(
     file_threshold: str = DEFAULT_FILE_THRESHOLD,
     path: Optional[Union[Path, str]] = None,
     initial: bool = False,
+    propagate: bool = False,
 ):
     """
     The initial set up is required since we have code (e.g. get_content_path) that runs in __main__ before the typer/click commands set up the logger.
-    In the initial set up there is NO file logging (only console)
+    In the initial set up there is NO file logging (only console).
+
+    Parameters:
+    - calling_function (str): The name of the function invoking the logger setup, included in all logs.
+    - console_threshold (str): Log level for console output.
+    - file_threshold (str): Log level for file output.
+    - path (Union[Path, str], optional): Path for file logs. If None, defaults to the calculated log directory.
+    - initial (bool): Indicates if the setup is for the initial configuration (console-only if True).
+    - propagate (bool): If True, propagates logs to Python's logging system.
+
     """
     global logger
     _setup_neo4j_logger()
 
-    logger.remove(None)  # Removes all pre-existing handlers
+    logger.remove()  # Removes all pre-existing handlers
```

Comment:
you sure passing None vs nothing still achieves what we wanted?

Reply:
Default value is None than yes, and it is less confusing that way imo. https://loguru.readthedocs.io/en/stable/api/logger.html#loguru._logger.Logger.remove